tests/test_input_damages.py

```diff
@@ -1356,3 +1381,166 @@ def test_error_coastal_inputs(
+    elif file_type == "wrong_batches":
+        # Create input data with wrong batches
+        batch_values = np.sort(
+            ["batch" + str(i) for i in np.arange(0, 14)]
+            + [
+                "batch1",
+            ]
+        )
+
+    if sector == "coastal":
+        data = np.ones(
+            (
+                len(ssp_values),
+                len(model_values),
+                len(slr_values),
+                len(batch_values),
+                len(year_values),
+                len(region_values),
+            )
+        )
+    else:
+        data = np.ones(
+            (
+                len(ssp_values),
+                len(rcp_values),
+                len(model_values),
+                len(slr_values),
+                len(batch_values),
+                len(year_values),
+                len(region_values),
+            )
+        )
+
+    # Create xarray dataset
+    if "coastal" in sector:
+        dims = ["ssp", "model", "slr", "batch", "year", "region"]
+        coords = {
+            "ssp": (["ssp"], ssp_values),
+            "model": (["model"], model_values),
+            "slr": (["slr"], slr_values),
+            "batch": (["batch"], batch_values),
+            "year": (["year"], year_values),
+            "region": (["region"], region_values),
+        }
+        chunkies = {
+            "ssp": 1,
+            "model": 1,
+            "slr": 1,
+            "batch": 5 if file_type == "wrong_chunk_sizes" else -1,
+            "year": 10,
+            "region": 3 if file_type == "wrong_region_chunk_sizes" else 6,
+        }
+    else:
+        dims = ["ssp", "rcp", "model", "gcm", "batch", "year", "region"]
+        coords = {
+            "ssp": (["ssp"], ssp_values),
+            "rcp": (["rcp"], rcp_values),
+            "model": (["model"], model_values),
+            "gcm": (["gcm"], slr_values),
+            "batch": (["batch"], batch_values),
+            "year": (["year"], year_values),
+            "region": (["region"], region_values),
+        }
+        chunkies = {
+            "ssp": 1,
+            "rcp": 1,
+            "model": 1,
+            "gcm": 1,
+            "batch": 5 if file_type == "wrong_chunk_sizes" else -1,
+            "year": 10,
+            "region": 3 if file_type == "wrong_region_chunk_sizes" else 6,
+        }
+
+    ds = xr.Dataset(
+        {
+            "data": (
+                dims,
+                data,
+            ),
+        },
+        coords=coords,
+    ).chunk(chunkies)
+
+    # Save xarray dataset as Zarr
+    ds.to_zarr(path, mode="w")
+
+
+@pytest.mark.parametrize("sector", ["mortality", "coastal"])
+def test_validate_damages_correct(tmp_path, sector):
+    path = str(tmp_path / f"damages_correct_{sector}.zarr")
+    file_type = "correct"
+    create_dummy_input_zarr(path, sector, file_type)
+    validate_damages(sector, path)  # No assertion error should be raised
+
+
+def test_validate_damages_incorrect_batches(tmp_path):
```

Comment:
Need docstr. What's this testing. I can kinda get a feel for it from the title but might want to say more specifically what you're looking for.

These docstrs also can print when tests fail so its good to have even if they seem obvious.